code/modules/mob/living/basic/turtle/turtle.dm

```diff
@@ -0,0 +1,240 @@
+#define PATH_PEST_KILLER "path_pest_killer"
+#define PATH_PLANT_HEALER "path_plant_healer"
+#define PATH_PLANT_MUTATOR "path_plant_mutator"
+#define REQUIRED_TREE_GROWTH 250
+#define UPPER_BOUND_VOLUME 50
+#define LOWER_BOUND_VOLUME 10
+
+/mob/living/basic/turtle
+	name = "turtle"
+	desc = "Dog."
+	icon_state = "turtle"
+	icon_living = "turtle"
+	icon_dead = "turtle_dead"
+	base_icon_state = "turtle"
+	icon = 'icons/mob/simple/pets.dmi'
+	butcher_results = list(/obj/item/food/meat/slab = 3, /obj/item/food/pickle = 1, /obj/item/stack/sheet/mineral/wood = 10)
```

Comment:
Why does it leave a pickle behind when butchered? 😅

Reply:
it was a silly reference to how they drop pickles on death in elden ring. but if its too out there i can remove it

Reply:
no no, it's good, I'm just don't play Elden Ring.